app/Models/MyList.php

```diff
@@ -9,19 +9,10 @@ class MyList extends Model
 {
     use HasFactory;
 
-    /**
-     * The table associated with the model.
-     *
-     * @var string
-     */
     protected $table = 'my_list';
 
-    /**
-     * The attributes that are mass assignable.
-     *
-     * @var array
-     */
     protected $fillable = [
```

Comment:
Please keep default comments, no need to remove.

Reply:
> Please keep default comments, no need to remove.

noted